app/models/manageiq/providers/amazon/cloud_manager/cloud_database.rb

```diff
@@ -44,6 +46,7 @@ def self.params_for_create
           :label        => _('Cloud Database'),
           :includeEmpty => true,
           :isRequired   => true,
+          :validate   => [{:type => 'required'}],
           :options      => ["aurora", "aurora-mysql", "aurora-postgresql", "mariadb", "postgres", "mysql", "oracle-ee", "oracle-ee-cdb", "oracle-se2", "oracle-se2-cdb", "sqlserver-ee", "sqlserver-se", "sqlserver-ex", "sqlserver-web"].map do |db|
```

Comment:
@GilbertCherrie Does this one suffer from the same drop-down problem we noticed elsewhere?

Reply:
It shouldn't, that drop down issue only affects selects that have `isSearchable: true` or `isClearable: true`.